### PR TITLE
feature/70_overwrite

### PR DESCRIPTION
**Pull request by Reagen**
Includes Issue #70

### CODE REVIEW

File: analysis/deep2_r23_o32/general.py [hunk 1]
```diff
@@ -137,8 +137,11 @@ def get_det3(fitspath, fitspath_ini, log=None):
...
     # used to be get_det3_table.tbl
     outfile = join(fitspath, filename_dict['indv_prop'])
-    log.info(f"Writing: {outfile}")
-    asc.write(tab1, outfile, format='fixed_width_two_line')
+    if not exists(outfile):
+        log.info(f"Writing: {outfile}")
+    else:
+        log.info(f"Overwriting: {outfile}")
+    asc.write(tab1, outfile, format='fixed_width_two_line', overwrite=True)
     # tab1.write(fitspath_ini+'get_det3_table.fit', format = 'fits', overwrite = True)
```
Comment (astrochun) [inline analysis/deep2_r23_o32/general.py:145]:
```suggestion
```

Comment (astrochun) [reply analysis/deep2_r23_o32/general.py:145]: Let's remove this unused line.

**Review by astrochun — CHANGES_REQUESTED** [review]
This looks good. Just one minor change.

**Review by Reagen — COMMENTED** [review]
Made the change

**Review by astrochun — APPROVED** [review]
Great. I'll go ahead and merge.

**Pull request merged**